packages/app/components/navbar/MobileMenuList.tsx

```diff
@@ -16,7 +16,7 @@ export default function MobileMenuList({ open, toggle }: MobileMenuListProps) {
         onClick={toggle}
       ></div>
       <div className="absolute left-0 w-full gap-2 px-6 py-2 border-b border-solid rounded-lg bg-surface-25 top-nav-height border-surface-75">
-        <Link href="#" className="flex items-center py-3 text-em-med">
+        <Link href="/stacks" className="flex items-center py-3 text-em-med">
```

Comment:
Should we toggle the menu when user clicks on a new route?

Reply:
Yes, we should

Reply:
Fixed.